conan/tools/apple/apple.py

```diff
@@ -1,10 +1,16 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-import os
 
 from conans.util.runners import check_output_runner
 
 
+def is_apple_cross_building(conanfile):
```

Comment:
Is this intended to be a public tool? I don't like it public, why the logic is not just embedded in the general ``cross_building()`` tool? It seems dirty that recipes would start to have custom code with ``is_apple_cross_building()``. Maybe let it be private to the toolchain at the moment?